test/functional/openapi_specification.yaml

```diff
@@ -0,0 +1,36 @@
+openapi: '3.0.3'
```

Comment:
This file should be in the directory or subdirectory of the test that uses it. It's ok if there are multiple copies of the same file.